zipline/utils/cli.py

```diff
@@ -99,9 +100,12 @@ def parse_args(argv, ipython_mode=False):
     parser.add_argument('--start', '-s')
     parser.add_argument('--end', '-e')
     parser.add_argument('--capital_base')
-    parser.add_argument('--source', choices=('yahoo',))
+    parser.add_argument('--source', '-d')
```

Comment:
This arg can now be a path to a file or folder, as well as 'yahoo', so I removed the choices.

Should I get rid of the single-letter entries? That may cause some user confusion, but I don't know how widely the CLI is actually used.